src/bittybuzz/bittybuzz.bst

```diff
@@ -1 +1,2 @@
+log
```

Comment:
In short, the bst file is to define string that are or are not in the script. 

First, this allows to have a string ID even if the string is not in the script. Since we use a string ID to register function (because bittybuzz refer to function with strings), this allows us to compile even if the string is not in the bzz script. 

For example. If we didn't have a .bst file and the .bzz script didn't contain log and logInt, we couldn't compile since the define BZZSTRID_log and BZZSTRID_logInt would not be defined in the C code.

Secondly is that our build uses the strings in the bst file and ignores putting them in memory. This allows us to only generate strings on symbols that we don't already know (i.e strings that are represented like this: "Hello world"). Saving space on the MCU and avoiding to have "logInt" as a string on the MCU.

idk if this was a clear explanation, it's a workaround to have string in the bbvm and avoiding the need to have ALL the buzz strings symbols (including functions name and global variable names) in the memory of the MCU

Reply:
yeah it's pretty much what I thought